tests/operator_test.go

```diff
@@ -233,7 +233,7 @@ var _ = Describe("Operator", func() {
 				return fmt.Errorf("Waiting for conditions to indicate update (conditions: %+v)", kv.Status.Conditions)
 			}
 			return nil
-		}, 120*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+		}, 300*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
```

Comment:
Can you please explain what is the reason for increasing the timeout and how it is related to this PR?

Reply:
This no longer happens, and as such I can remove this (highly suspicious) timeout increase.